fe/fe-core/src/main/java/com/starrocks/sql/analyzer/AlterTableStatementAnalyzer.java

```diff
@@ -393,6 +393,8 @@ public Void visitAddColumnsClause(AddColumnsClause clause, ConnectContext contex
             if (columnDefs == null || columnDefs.isEmpty()) {
                 throw new SemanticException("Columns is empty in add columns clause.");
             }
+            boolean hasMaterializedColumn = false;
```

Comment:
You should change all the `materializedColumn` to `generatedColumn` in the codepaths.